src/qmflows/utils.py

```diff
@@ -53,31 +55,52 @@ def dict2Setting(d):
     return r
 
 
-def init_restart(path: Union[None, AnyStr, os.PathLike] = None,
-                 folder: Union[None, AnyStr, os.PathLike] = None) -> None:
+def init_restart(path: Union[None, str, os.PathLike] = None,
+                 folder: Union[None, str, os.PathLike] = None,
+                 load_jobs: bool = False) -> None:
```

Comment:
Do you feel the default value of this agument should be ``True``?
Automatically loading previous jobs is nice, but the pickling process can be quite slow.

Reply:
It depends on the size of the jobs that are most commonly run. It the most common jobs are significantly large then I think we should make it `False` by default

Reply:
Indeed, I feel it's definitely the safer option of the two.